dask-gateway/dask_gateway/client.py

```diff
@@ -243,10 +255,23 @@ class Gateway(object):
     loop : IOLoop, optional
         The IOLoop instance to use. Defaults to the current loop in
         asynchronous mode, otherwise a background loop is started.
+    http_client: str, optional
+        Short-name for which http backend to use with Tornado. Either ``curl``
+        or ``simple``. Default is ``curl`` if ``pycurl`` is installed.
+    client_defaults: dict, optional
+        Configuration dictionary to pass to http backend. Needs to be compatible
+        with chosen ``http_client``
     """
 
     def __init__(
-        self, address=None, proxy_address=None, auth=None, asynchronous=False, loop=None
+        self,
+        address=None,
+        proxy_address=None,
+        auth=None,
+        asynchronous=False,
+        loop=None,
+        http_client=None,
+        client_defaults=None,
```

Comment:
Hmmm, I hadn't thought about kwarg names for these. These read a bit off to me. Perhaps:

```
httpclient_type=None,
httpclient_defaults=None
```

to better match the config names?


As written `http_client` looks like it expects an http client object, and `client_defaults` makes me ask "which client".